devtools-ci-index.yaml

```diff
@@ -1673,7 +1673,7 @@
     secret-values:
       - env-var: 'QUAY_SECRET_JSON'
         vault-key: 'secret'
-
+        
```

Comment:
minor pet peeve here, can you remove the indented spaces? otherwise LGTM

Reply:
These sections are usually divided by empty row so I though it was a kind of "mistake" that there is empty row missing. It is more readable now. WDYT? If you don't like this change I can revert it.

Reply:
I agree there should be an empty line separating the sections, the problem is that in your commit it isn't really empty, it's a line that contains only 4 spaces. I run a `grep` to see if this was an issue elsewhere in the doc, but this is the only line in which this happens.

I'd like it if you could keep the new line separting the sections, but remove those 4 spaces.

```
$ git checkout -b Katka92-rhche_add_gh_token master
Switched to a new branch 'Katka92-rhche_add_gh_token'

$ git checkout -b Katka92-rhche_add_gh_token master
Switched to a new branch 'Katka92-rhche_add_gh_token'
openshiftio-cico-jobs git:(Katka92-rhche_add_gh_token) ✗ $ git pull https://github.com/Katka92/almighty-jobs.git rhche_add_gh_token
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 3 (delta 2), reused 1 (delta 0), pack-reused 0
Unpacking objects: 100% (3/3), done.
From https://github.com/Katka92/almighty-jobs
 * branch            rhche_add_gh_token -> FETCH_HEAD
Updating 0d6bfe3..cb0d780
Fast-forward
 devtools-ci-index.yaml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

$ grep -n -E '^ +$' devtools-ci-index.yaml
1676:        
```